jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetailsmulti/TaskDetailsMultiPresenter.java

```diff
@@ -135,6 +135,8 @@ public void setIsForLog(boolean isForLog) {
     }
 
     public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
+        boolean resetTabs = (taskId != event.getTaskId());
```

Comment:
@nmirasch this comparison seems to break findbugs, one of the reasons is that comparing with a _!=_ could throw a NPE.